test/integration/Android/ElementTest.cs

```diff
@@ -1,8 +1,10 @@
 ﻿using Appium.Net.Integration.Tests.helpers;
```

Comment:
I would be happy to see more unit tests, which cover most of the builder methods. This stuff could easily be tested in isolation.

Reply:
Will do. Is it enough to validate that the correct string is built, or would you prefer a full integration test where the built statement is used to find an element?

Reply:
the first one is pretty enough for a unit test

Reply:
Unit tests added. Feels a little weird testing strings, but it did uncover one incorrect builder statement. Let me know if more needs to be done.

Reply:
> Feels a little weird testing strings, but it did uncover one incorrect builder statement

And this is great